config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -16,9 +18,14 @@ presubmits:
       containers:
       - args:
         - --timeout=70
-        - --bare
+        - --root=/go/src
+        - --repo=k8s.io/kubernetes=$(PULL_REFS)
+        - --repo=k8s.io/release
+        - --upload=gs://kubernetes-jenkins/pr-logs
         - --scenario=kubernetes_e2e
         - --
+        - --build=bazel
+        - --extract=local
```

Comment:
this is duplicate now

Reply:
fixed (removed the bottom one, so that arg ordering is more similar to the ipvs job were copying)